docs/advanced_topics/customisation/page_editing_interface.rst

```diff
@@ -61,6 +61,49 @@ However, template output from :class:`~wagtail.wagtailcore.fields.RichTextField`
 
 If you're interested in extending the capabilities of the Wagtail WYSIWYG editor (``hallo.js``), See :ref:`extending_wysiwyg`.
 
+.. _custom_config:
+
+Passing a custom ``hallo.js`` editor configuration
+--------------------------------------------------
+
+A custom ``hallo.js`` JSON configuration can be passed through the default widgets RichTextArea ``editor_config`` attribute.
+
+.. code-block:: python
+
+    from wagtail.wagtailcore.fields import RichTextField
+    from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+    from wagtail.wagtailcore.fields import RichTextArea
+
+    simple_config={
+        'halloheadings': {
```

Comment:
The page-editor now by default implements and handles a hallo configuration instead of only hallo plugin. 

I have updated the pull requests codebase to reflect these changes.